Composer/packages/extensions/adaptive-form/src/components/fields/NumberField.tsx

```diff
@@ -36,7 +37,7 @@ const NumberField: React.FC<FieldProps> = (props) => {
     onChange(newValue);
   };
 
-  const step = type === 'integer' ? 1 : 0.1;
+  const step = type === 'integer' ? 1 : 1 / Math.pow(10, floatNumberOfDecimals);
```

Comment:
This could just be `Math.pow(10, -floatNumberOfDecimals)`.

Reply:
Good point!